docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -190,4 +190,32 @@ Scenario: Verify step When I wait until HTTP $httpMethod request with URL patter
 Given I am on a page with the URL 'http://httpbin.org/get'
 When I wait until HTTP GET request with URL pattern `http://httpbin.org/get` exists in proxy log
 Then number of HTTP GET requests with URL pattern `http://httpbin.org/get` is EQUAL TO `1`
+----
+
+
+=== Gets a list of elements by locator and check that it is sorted by text with using sortingOrder by element text
+
+==== *_Info_*
+Gets a list of elements by locator and check that it is sorted by text with using sortingOrder by element text. List should have more than 1 elements.
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+Then elements located `$locator` are sorted by text in $sortingOrder order
+----
+
+==== *_Parameters_*
+
+. `$locator` - to locate elements
```

Comment:
please create separate page in docs for search attributes and refer there
cc @valfirst please advice

Reply:
@uarlouski I think that's an overhead for this PR